octobot_backtesting/collectors/exchanges/exchange_collector.py

```diff
@@ -44,8 +44,28 @@ def __init__(self, config, exchange_name, tentacles_setup_config, symbols, time_
         self.use_all_available_timeframes = use_all_available_timeframes
         self.start_timestamp = start_timestamp
         self.end_timestamp = end_timestamp
+        self.finished = False
+        self.in_progress  = False
```

Comment:
I think we should add those two attributes to `DataCollector` class instead of `ExchangeDataCollector`